src/objects/token/admin.py

```diff
@@ -14,7 +14,19 @@
 
 @admin.register(Permission)
 class PermissionAdmin(admin.ModelAdmin):
-    list_display = ("token_auth", "object_type", "mode", "use_fields")
+    list_display = (
+        "token_auth",
+        "object_type",
+        "mode",
+        "use_fields",
+        "get_uuid",
+    )
+    readonly_fields = ("get_uuid",)
```

Comment:
We rewrite the template for the token admin page with react, so you need also to add the new field in the `permission-form.js` file

Reply:
This wasn't asked for in the ticket so lets ignore it for now.

Reply:
In this case please remove `get_uuid` from readonly_fields, because it's not shown in the detailed page

Reply:
I think this is not yet addressed?